core/options.c

```diff
@@ -2404,6 +2404,13 @@ options_exit()
     DELETE_READWRITE_LOCK(options_lock);
 }
 
+/* reset dynamo options to defaults */
+void
+options_detach()
```

Comment:
This PR as-is does not actually solve the problem I'm hitting in my test: I'm using raw2trace in between a detach and attach and it doesn't call this function.  Maybe add a call to options_detach() in standalone_exit()?  I confirmed that solves it for me.  (Or bite the bullet and pay for this on every options_exit().)

Reply:
Done, put it in standalone_exit